src/main/app/src/app/informatie-objecten/model/enkelvoudig-informatie-object-zoek-parameters.ts

```diff
@@ -8,4 +8,5 @@ export class EnkelvoudigInformatieObjectZoekParameters {
     zaakUUID: string;
     UUIDs: string[];
     toonGekoppeldeZaakDocumenten: boolean;
+    toegestaneInformatieObjectTypeUUIDs: string[];
```

Comment:
Hier kun je beter een Besluit mee geven, omdat je wilt weten welke documenten je mag toevoegen aan de hand van een besluit. Dus die filtering hoort in de backend te gebeuren.

Reply:
Opgelost door besluittype UUID mee te geven in opvolgende commit